### ISSUE TITLE
Some tests fail on FreeBSD

### ISSUE THREAD

**Issue by yurivict** [mlpack/ensmallen#264]
#### Issue description
```

$ ./ensmallen_tests
ensmallen version: 2.16.1 (Severely Dented Can Of Polyurethane)
armadillo version: 10.1.1 (Orchid Ambush)

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
ensmallen_tests is a Catch v2.4.1 host application.
Run with -? for options

-------------------------------------------------------------------------------
CNESchafferFunctionN4Test
-------------------------------------------------------------------------------
/usr/ports/math/ensmallen/work/ensmallen-2.16.1/tests/cne_test.cpp:122
...............................................................................

/usr/ports/math/ensmallen/work/ensmallen-2.16.1/tests/cne_test.cpp:141: FAILED:
  REQUIRE( coordinates(0) == Approx(0).margin(0.1) )
with expansion:
  1.2531376349 == Approx( 0.0 )

===============================================================================
test cases:   274 |   273 passed | 1 failed
assertions: 13330 | 13329 passed | 1 failed
```

#### Your environment

 * version of ensmallen: 2.16.1
 * operating system: FreeBSD 12.2
 * compiler: clang-10
 * version of Armadillo: 10.1.1

**Comment by rcurtin** [comment]
It looks like you have run the tests from the wrong directory or something---the majority of those issues appear to be that the data files can't be loaded.

**Comment by yurivict** [comment]
Sorry, I first pasted a wrong log.

There is only one failure:
```
tests/cne_test.cpp:141: FAILED:
  REQUIRE( coordinates(0) == Approx(0).margin(0.1) )
with expansion:
  1.2531376349 == Approx( 0.0 )
```

**Comment by rcurtin** [comment]
I'm not sure but this could be related to https://github.com/mlpack/ensmallen/pull/267.

**Comment by rcurtin** [comment]
Actually I'm pretty sure this was fixed with #267, so I'll go ahead and close this.  Hopefully, next release, everything will pass for you. :)
